src/VivViewer.js

```diff
@@ -4,6 +4,17 @@ import { OrthographicView } from '@deck.gl/core';
 import { VivViewerLayer, StaticImageLayer } from './layers';
 import VivViewerOverview from './VivViewerOverview';
 
+// Taken from https://stackoverflow.com/a/31732310/8060591
+function checkIsSafari() {
```

Comment:
```suggestion
function isSafari() {
```
Preference for using `is` or `has` etc. when it returns a boolean.

Reply:
Sounds good.